Azure/sonic-buildimage
# Convert docker-teamd to buster

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>

**- Why I did it**
This is an enhancement.

**- How I did it**
Code change

**- How to verify it**
Build `target/docker-teamd.gz`
Verify that `/etc/apt/sources.list` points to `buster` using ` docker exec teamd cat /etc/apt/sources.list`

LACP portchannel comes UP.
```
root@sonic:~# show interface portchannel
Flags: A - active, I - inactive, Up - up, Dw - Down, N/A - not available,
       S - selected, D - deselected, * - not synced
  No.  Team Dev       Protocol     Ports
-----  -------------  -----------  -------------
   10  PortChannel10  LACP(A)(Up)  Ethernet84(S)
root@sonic:~# 
```

**- Description for the changelog**
Convert docker-teamd to buster


**- A picture of a cute animal (not mandatory but encouraged)**
